index.rst

```diff
@@ -160,6 +160,7 @@ Core
     Integration, components/sensor/integration, sigma.svg
     MQTT Subscribe, components/sensor/mqtt_subscribe, mqtt.png
     Uptime Sensor, components/sensor/uptime, timer.svg
+    Duty Time, components/sensor/duty_time, timer-play-outline.svg
```

Comment:
This probably should go under `miscellaneous`

Reply:
Added to this category by analogy with the `Uptime` component. I don't see a fundamental difference. Why is `Uptime` in the `Core` category?

Reply:
Jesse, here I will go on principle, or we will move all the controversial components `Uptime`, `Integration`, `WiFi Signal` to the category `miscellaneous` or leave it as it is. I'm sorry, but I don't see the difference between them. Why are they in this category? Should there be some kind of logic? :)

Reply:
In general, at first I was indignant that `Uptime` was also in the `core` category, by analogy with which I added my component here. But then I decided to take the `Integration` component into the `miscellations` too.